content/rest-api/resources.md

```diff
@@ -1110,6 +1110,79 @@ Below is the JSON format representing a reference entity.
 ::: panel-link Want more details about the reference entity resource? [Check its endpoints here!](/api-reference.html#Referenceentity)
 :::
 
+### Reference entity attribute (3.x and EE only)
+
+Reference entities have attributes that allow to describe their [records](/documentation/resources.html#reference-entity-record-3x-and-ee-only).
+
+::: warning
+Note that "Reference entity attribute" are not the same entity than "Attribute", even though they have a lot in common. The first one is used to describe reference entities records, the second one is use to describe products.
+:::
+
+In the Akeneo UI, you can find the attributes that compose a given reference entity by following these simple steps. Go in the `Entities` menu. Click on a reference entity and then, click on the `Attributes` sub menu. Below is an example of the attributes that compose the `Designer` entity in the UI.
+
+![Reference entity attributes](/img/screenshots/v3.0/reference_entity_attribute.png)
+
+Below is the JSON standard format representing a reference entity attribute.
+
+```json
+{
+  "code": "description",
+  "labels": {
+    "en_US": "Description",
+    "fr_FR": "Description"
+  },
+  "type": "text",
+  "localizable": true,
+  "scopable": false,
+  "max_characters": null,
+  "is_textarea": true,
+  "is_rich_text_editor": true,
+  "validation_rule": null,
+  "validation_regexp": null
+}
+```
+
+::: panel-link Want more details about the reference entity attribute resource? [Check its endpoints here!](/api-reference.html#Referenceentityattribute)
+:::
+
+### Reference entity attribute option (3.x and EE only)
+
+Some type of attributes of reference entities can offer list of choices. These available choices are reference entity attribute options.
+
+::: warning
+Note that "Reference entity attribute option" are not the same entity than "Attribute option", even though they have a lot in common. The first one is an option of an attribute used to describe reference entities records, the second one is an option of an attribute used to describe products.
+:::
+
+Only attribute of type simple select or multiselect can have options.
+
+In the Akeneo UI, ... ?? TODO
```

Comment:
The UI part has been done, you should be able to complete this.

Reply:
:+1: